test/avocado/selenium-base.py

```diff
@@ -63,42 +63,73 @@ def test50ChangeTabLogs(self):
         self.wait_id("journal-current-day")
         self.wait_id("journal-prio")
         self.click(self.wait_xpath(
-            "//span[@id='journal-prio' and contains(text(), '%s')]" % "Problems, Errors"))
+            "//span[@id='journal-prio' and contains(text(), '%s')]" % "Error and above"))
```

Comment:
Hmm, how does that work with `translatable="yes"` in the HTML? I mean wouldn't that `"Error and above"` text be replaced with the translation, breaking this match?

Reply:
Tests run under C.utf8 (except for tests like check-pages which change the language explicitly), so that should be fine.